dashboards-observability/.cypress/integration/app_analytics.spec.js

```diff
@@ -296,36 +315,38 @@ describe('Viewing application', () => {
     cy.get('.euiBadge__text').contains('Base Query').should('exist');
   });
 
-  it('Saves visualization to panel', () => {
-    cy.get('.euiTab').contains('Log Events').click();
+  it('Saves visualization #1 to panel', () => {
+    cy.get('.euiTab').contains('Panel').click();
+    cy.get('.euiButton').eq(3).contains('Add').click();
+    cy.wait(delay);
     cy.get('.plot-container').should('exist');
     cy.get('[data-test-subj="searchAutocompleteTextArea"]').click();
     cy.get('.aa-List').find('.aa-Item').should('have.length', 11);
     cy.get('[data-test-subj="searchAutocompleteTextArea"]').type('x');
-    cy.get('[data-test-subj="searchAutocompleteTextArea"]').clear().wait(delay * 2).type(spanQueryPartOne);
-    cy.get('[data-test-subj="searchAutocompleteTextArea"]').wait(delay * 2).type(spanQueryPartTwo);
-    cy.get('[data-test-subj="searchAutocompleteTextArea"]').wait(delay * 2).type(spanQueryPartThree);
-    changeTimeTo24('hours');
-    cy.get('.euiButton').contains('Refresh').click();
+    cy.get('[data-test-subj="searchAutocompleteTextArea"]').clear().wait(delay * 2).type(spanQueryOnePartOne);
+    cy.get('[data-test-subj="searchAutocompleteTextArea"]').wait(delay * 2).type(spanQueryOnePartTwo);
+    cy.get('[data-test-subj="searchAutocompleteTextArea"]').wait(delay * 2).type(spanQueryOnePartThree);
+    changeTimeTo24('months');
     cy.get('.euiTab').contains('Visualizations').click();
     supressResizeObserverIssue();
     cy.get('[data-test-subj="eventExplorer__saveManagementPopover"]').click();
-    cy.get('[data-test-subj="eventExplorer__querySaveName"]').click().type(visName);
+    cy.get('[data-test-subj="eventExplorer__querySaveName"]').click().type(visOneName);
+    cy.wait(delay);
     cy.get('[data-test-subj="eventExplorer__querySaveConfirm"]').click();
     cy.wait(delay);
     cy.get('.euiTab').contains('Panel').click();
     cy.wait(delay);
     cy.get('.js-plotly-plot').should('exist');
   });
 
-  it('Adds availability level to visualization', () => {
+  it('Adds availability level to visualization #1', () => {
     cy.get('.euiTab').contains('Panel').click();
     cy.wait(delay);
     cy.get('[aria-label="actionMenuButton"]').click();
     cy.get('.euiContextMenuItem').contains('Edit').click();
     supressResizeObserverIssue();
     cy.wait(delay);
-    cy.get('[data-test-subj="superDatePickerShowDatesButton"]').should('contain', 'Last 24 hours');
+    cy.get('[data-test-subj="superDatePickerShowDatesButton"]').should('contain', 'Last 24 months');
```

Comment:
will this test work when it's 2024?

Reply:
The year 2024??

Reply:
Do the sample logs not get updated every time the cypress tests run??

Reply:
if it's using dashboards sample data then should be ok, if using the trace data then no

Reply:
Most of the tests with time range changes are using dashboards sample data with logs so I think it should be okay.